### PR TITLE
Implement operations manager

### PR DESCRIPTION
**Pull request by kaloyanspiridonov**
**Description**

Changes proposed in this pull request:
- Introduce new operation-manager component

**Related issue(s)**

- ...

**Pull Request status**

- [x] Implementation
- [x] Unit tests
- [ ] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/internal/operations_manager/opcreator.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+			operations = append(operations, buildORDOperationInput(data))
+		}
+	}
+
+	return operations, nil
+}
+
+func (oc *ORDOperationCreator) appTemplateWebhookToOperations(ctx context.Context, webhook *model.Webhook) ([]*model.OperationInput, error) {
+	operations := make([]*model.OperationInput, 0)
+	if webhook.ObjectType != model.ApplicationTemplateWebhookReference {
```
Comment (alextargov) [inline components/director/internal/operations_manager/opcreator.go:84]: We've already did a similar check on L63. Do we really need this one too?

File: components/director/internal/operations_manager/opcreator.go [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+		return errors.Wrap(err, "while creating multiple operations")
+	}
+
+	return tx.Commit()
+}
+
+func (oc *ORDOperationCreator) buildOperationInputs(ctx context.Context) ([]*model.OperationInput, error) {
+	ordWebhooks, err := oc.getWebhooksWithOrdType(ctx)
+	if err != nil {
+		return nil, errors.Wrap(err, "while getting webhooks of type OPEN_RESOURCE_DISCOVERY")
```
Comment (alextargov) [inline components/director/internal/operations_manager/opcreator.go:58]: Could we parametrize the `OPEN_RESOURCE_DISCOVERY` like it is on L115

File: components/schema-migrator/migrations/director/20230505102050_create_operation.up.sql [hunk 3]
```diff
@@ -0,0 +1,15 @@
+BEGIN;
```
Comment (alextargov) [inline components/schema-migrator/migrations/director/20230505102050_create_operation.up.sql:1]: This [PR](https://github.com/kyma-incubator/compass/pull/3066) introduces a new migration. By the time we merge this change, the PR may have been already merged. This is a reminder to check the migration name before merging this change.

File: components/director/internal/operations_manager/opcreator.go (hunk 1 above)
Comment (kaloyanspiridonov) [reply components/director/internal/operations_manager/opcreator.go:84]: in case someone decide to reuse this method

**Review by alextargov — APPROVED** [review]

**Pull request merged**